README.md

```diff
@@ -21,68 +20,70 @@ A minimalistic Flutter game engine.
-
-We provide many examples of different features which can be tried out from your browser
-[here](https://flame-engine.github.io/flame/). You can also check the examples source code [here](https://github.com/flame-engine/flame/tree/main/examples).
-
-The official site for Flame, which also contains the documentation can be found
-[here](https://flame-engine.org/).
 
 ## Help
 
-We have a Flame help channel on Blue Fire's Discord, join it [here](https://discord.gg/5unKpdQD78).
-
-We also have a [FAQ](FAQ.md), so please search for your question there first.
-
-## Goals
-
-The goal of this project is to provide a complete set of out-of-the-way solutions for the common
-problems every game developed in Flutter will share.
+We have a Flame community and help channel on [Blue Fire's Discord](https://discord.gg/5unKpdQD78).
+
+If you are more comfortable with StackOverflow you can create a question with the
+[Flame tag](https://stackoverflow.com/questions/tagged/flame) and we'll reply as soon as we can.
+
+
+## Features
+
+The goal of Flame is to provide a complete set of out-of-the-way solutions for the common problems
+every game developed in Flutter will share.
+
+Currently Flame provides:
+ - A game loop.
+ - A component/object system (FCS).
+ - Effects and particles.
+ - Collision detection.
+ - Gesture and input support.
+ - Images, animations, sprites and sprite sheets.
+ - Other utilities to make development easier.
+
+On top of those features we offer something that we call bridge libraries. Through these libraries
+we offer Flame components and bindings to other packages. Currently we have bridge
+libraries to the following packages:
+
+- [flame_audio](https://github.com/flame-engine/flame/tree/main/packages/flame_audio) for
+  [AudioPlayers](https://github.com/bluefireteam/audioplayers): Play multiple audio files simultaneously.
+- [flame_bloc](https://github.com/flame-engine/flame/tree/main/packages/flame_bloc) for
+  [Bloc](https://github.com/felangel/bloc): A predictable state management library.
+- [flame_fire_atlas](https://github.com/flame-engine/flame/tree/main/packages/flame_fire_atlas) for
+  [FireAtlas](https://github.com/flame-engine/fire-atlas): Create texture atlases for games.
+- [flame_forge2d](https://github.com/flame-engine/flame/tree/main/packages/flame_forge2d) for
+  [Forge2D](https://github.com/flame-engine/forge2d): A Box2D physics engine.
+- [flame_lint](https://github.com/flame-engine/flame/tree/main/packages/flame_forge2d) -
+  Our set of linting (`analysis_options.yaml`) rules.
+- [flame_oxygen](https://github.com/flame-engine/flame/tree/main/packages/flame_oxygen) for
+  [Oxygen](https://github.com/flame-engine/oxygen): A lightweight Entity Component System (ECS) framework.
+- [flame_rive](https://github.com/flame-engine/flame/tree/main/packages/flame_rive) for
+  [Rive](https://rive.app/): Create interactive animations.
```

Comment:
missed flame_flare!

Reply:
But do we want people to use that, don't we want them to use Rive?